docs/pages/includes/common-faq.mdx

```diff
@@ -0,0 +1,25 @@
+{{ level="##" }}
```

Comment:
I think the `level` param is clever, though it would be more useful if there were multiple subheadings in the middle of the partial and any page that included the partial _had_ to include those subheadings.

In this case, though, since the heading is at the top of the partial and there is only one, I think it would make sense to remove the heading from this partial and add it to the two pages that include the partial instead.